x-pack/plugins/siem/public/components/page/network/domains_table/first_last_seen/index.test.tsx

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { cloneDeep } from 'lodash/fp';
+import * as React from 'react';
+import { MockedProvider } from 'react-apollo/test-utils';
+import { render } from 'react-testing-library';
+
+import { mockFirstLastSeenDomainQuery } from '../../../../../containers/domains/first_last_seen_domain/mock';
+import { FlowTarget } from '../../../../../graphql/types';
+import { wait } from '../../../../../lib/helpers';
+import { TestProviders } from '../../../../../mock';
+
+import { FirstLastSeenDomain } from '.';
+
+describe('FirstLastSeen Component', async () => {
```

Comment:
These tests mirror @XavierM's from https://github.com/elastic/kibana/pull/34831 -- this implementation of hooks for `firstSeen`/`lastSeen` should only be temporary, so we opted to not abstract this out into a common component. If they end up sticking around and calling the codebase home then we should consolidate towards a component with common behavior.